src/ZodError.ts

```diff
@@ -40,6 +41,16 @@ export class ZodError extends Error {
     ]);
   };
 
+  static fromObject = (details: object) => {
+    return ZodError.create([
+      {
+        path: [],
```

Comment:
is path always equals to `[]`? 
Or is it updated somewhere else?

Reply:
this attribute is filling differently so initialisation looks correctly